packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -83,10 +83,13 @@ export interface IContainerRuntime
 	 * only after context.attachRuntime() is called.
 	 * @param pkg - package path
 	 * @param rootDataStoreId - data store ID (unique name). Must not contain slashes.
+	 * @param groupId - group to which this data stores belongs to. This is also known at service side and can be used to
+	 * fetch snapshot contents like snapshot tree, blobs using this id from the storage.
 	 */
 	createDetachedRootDataStore(
 		pkg: Readonly<string[]>,
 		rootDataStoreId: string,
+		groupId?: string,
 	): IFluidDataStoreContextDetached;
```

Comment:
It would be great to deprecate all "root" APIs and not invest new features into them (thus create more pressure for people to move to other APIs).

I do not remember why this API exists in the first place, but I'd think that with all the work around aliasing, including latest @tyler-cai-microsoft's changes / proposal, we should eliminate all "root" APIs, and instead on care about "regular" API with aliasing on top of it.

Reply:
oh ok. So, I would remove remove groupID from the Root datastore one if its not needed. I added it on here as this api is not yet deprecated. For example, _createDataStoreWithProps api was deprecated and so I didn't added groupid to that api.

Reply:
Removed.